src/utils/input/utils.ts

```diff
@@ -82,7 +82,9 @@ export const getStdIn = ({
 				dataIsRequired
 			)
 		) {
-			return resolve({});
+			resolve({});
```

Comment:
`return resolve({})` was fine I guess or was there any reason?

Reply:
tslint rule, where if `void` is return type, we shouldn't return value, and explicitly do return in new line